src/TriggerActivityMakerChannelAdjacency.cpp

```diff
@@ -75,8 +87,13 @@ TriggerActivityMakerChannelAdjacency::operator()(const TriggerPrimitive& input_t
 	ta_found = 1;
 	m_ta_count++;
 	if (m_ta_count % m_prescale == 0){
-	  
-	  output_ta.push_back(construct_ta(win_adj_max));
+	  auto ta = construct_ta(win_adj_max);
+	  output_ta.push_back(ta);
+          // Update OpMon Variable(s)
+          system_time = duration_cast<milliseconds>(system_clock::now().time_since_epoch()).count();
+	  data_time = ta.time_start*(16*1e-6);
+          m_data_vs_system_time_out.store(fabs(system_time - data_time - m_initial_offset)); // Store the difference for OpMon
+
```

Comment:
Might be worth to check the latencies before and after this change. `auto ta` followed by `push_back` will allocate extra memory somewhere, which will slow things down and increase memory usage (not by much, I'm more worried about the slowdown than extra memory).

If it's only used for `ta.time_start*(magic)` then maybe we could do something like `output_ta.back().time_starrt*(magic)`, since whatever we pushback is always at the end?